forge-gui/res/cardsfolder/upcoming/the_flood_of_mars.txt

```diff
@@ -0,0 +1,12 @@
+Name:The Flood of Mars
+ManaCost:2 U U
+Types:Creature Alien Zombie Horror
+PT:3/3
+K:Islandwalk
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ DBCounter | TriggerDescription$ Water Always Wins — Whenever CARDNAME attacks, put a flood counter on another target creature or land. If it's a creature, it becomes a copy of CARDNAME. If it's a land, it becomes an Island in addition to its other types.
+SVar:DBCounter:DB$ PutCounter | ValidTgts$ Creature.Other,Land.Other | AITgts$ Creature.OppCtrl+notnamedThe Flood of Mars,Land.OppCtrl+nonIsland | TgtPrompt$ Select target creature or land | CounterType$ FLOOD | SubAbility$ DBCopy
+SVar:DBCopy:DB$ Clone | ConditionDefined$ ParentTarget | ConditionPresent$ Creature | Defined$ Self | CloneZone$ Battlefield | CloneTarget$ ParentTarget | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | ConditionDefined$ ParentTarget | ConditionPresent$ Land | Defined$ ParentTarget | Type$ Island | Duration$ Permanent | IsCurse$ True

```

Comment:
I've tried so many things by now, but for whatever reason, this effect refuses to turn the targeted land into an Island. I think this may be beyond my power to resolve.

Reply:
don't worry, fixed the typo for you

Reply:
What was the issue?

Reply:
check last commit to this branch for diff views

Reply:
Well, that was embarrassing.

Reply:
The s or not before the `$` can be tough to catch

When you start to actually debug it becomes easier to narrow down where the mistakes must be

Hang in there, it's great to have another contributor